### ISSUE TITLE
HTML2canvas SVG graph IE11

### ISSUE THREAD

**Issue by ArjunMahadik** [niklasvh/html2canvas#1426]
I am using Newest version of html2canvas ("html2canvas": "^1.0.0-alpha.9") to convert HTML as well as SVG graphs. It is working fine with chrome but not working on IE 11. Is there any need to handle SVG graphs?

**Comment by AKSET** [comment]
I am also facing the same issue. 
In chrome everything seems fine. In IE 10/11, for pie charts, the conversion is successfully but for bar graph, I am seeing black background.

**Comment by Mani2693** [comment]
Same issue here , Please help to fix it

Thanks 
Mani

**Comment by reinerluke97** [comment]
Doesnt work for me in newest Versions.

**Comment by nettiopsu** [comment]
Yes, there is a bug. Both inline and in image SVG files are not displayed in html2canvas in IE11, it seems to be removed from canvas at all (?), as it is not appearing even as a black box.

One solution here could be transforming SVG to PNG through Canvas, as PNG is still working.

**Comment by prestonj** [comment]
Ditto - 1.0.0 alpha 12 doesn't seem to display highcharts graphs (5.0.2) in IE11.

0.5.0 beta 4 was displaying corrupt graphs in IE11. Pie charts were warped and labels were duplicated.

**Comment by arielgelbard** [comment]
my TypeScript solution to make svgs render onto the screenshot:
`
import { Canvg } from 'canvg';
...
private convertSvgD3ChartsToPngs(): HTMLImageElement[] {
        const pngD3Charts: HTMLImageElement[] = [];

Array.from(document.getElementsByTagName('svg')).map(svg => {
            let svgD3ChartCanvas = document.createElement('canvas');

            // Increase the Pixel Density for the Screenshot
            svgD3ChartCanvas.width = parseInt(svg.getAttribute('width')) * 4;
            svgD3ChartCanvas.height = parseInt(svg.getAttribute('height')) * 4;

            let canvasContext = svgD3ChartCanvas.getContext('2d');

            // Use Canvg to convert the svg into a png
            let convertedSvgToPng = Canvg.fromString(
                canvasContext,
                (svg.parentNode as HTMLElement).innerHTML,
                {
                    ignoreDimensions: true,
                    scaleWidth: svgD3ChartCanvas.width,
                    scaleHeight: svgD3ChartCanvas.height
                }
            );
            convertedSvgToPng.start();

            // Attach new png
            let pngD3Chart = new Image();
            pngD3Chart.src = svgD3ChartCanvas.toDataURL('image/png');
            pngD3Chart.style.width = '100%';
            pngD3Charts.push(pngD3Chart);

            // Remove HTML Attribute and use CSS
            (pngD3Chart as any).width = '';
            svg.parentNode.parentNode.appendChild(pngD3Chart);
            (svg.parentNode as HTMLElement).style.visibility = 'hidden';
            (svg.parentNode as HTMLElement).style.height = '0';
            (svg.parentNode as HTMLElement).style.minHeight = '0';
        });
        return pngD3Charts;
    }`
